Tests/Common/Orders/Fees/FTXFeeTests.cs

```diff
@@ -0,0 +1,104 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using NUnit.Framework;
+using QuantConnect.Data;
+using QuantConnect.Data.Market;
+using QuantConnect.Orders;
+using QuantConnect.Orders.Fees;
+using QuantConnect.Securities;
+using QuantConnect.Securities.Crypto;
+using System;
+
+namespace QuantConnect.Tests.Common.Orders.Fees
+{
+    [TestFixture]
+    public class FTXFeeTests
+    {
+        private Crypto _xrpusdt;
+        private Crypto _ethusd;
+        private readonly IFeeModel _feeModel = new FTXFeeModel();
+
+        [SetUp]
+        public void Initialize()
+        {
+            var tz = TimeZones.NewYork;
+            _xrpusdt = new Crypto(
+                SecurityExchangeHours.AlwaysOpen(tz),
+                new Cash("USDT", 0, 1),
+                new SubscriptionDataConfig(typeof(TradeBar), Symbol.Create("XRP/USDT", SecurityType.Crypto, Market.FTX), Resolution.Minute, tz, tz, true, false, false),
+                new SymbolProperties("XRP/USDT", "USDT", 1, 0.01m, 0.00000001m, string.Empty),
```

Comment:
>  new SymbolProperties

Let's replace this for fetching the actual SP from the DB, same bellow 👍 

> TimeZones.NewYork;

Should really be UTC here?